tee-worker/app-libs/stf/src/trusted_call.rs

```diff
@@ -718,6 +722,12 @@ where
 				)));
 				Ok(())
 			},
+			TrustedCall::set_scheduled_mrenclave(_, bn, mrenclave) => {
```

Comment:
It's slightly risky to leave out the first account checker, although we know (internally) it's used for dev purposes only. Imagine the code is accidentally published to a production enclave.

I suggest adding at least `is_root` checker, but this is a minor problem.

Reply:
done